src/ui/src/app/chart-details/chart-details-versions/chart-details-versions.component.html

```diff
@@ -1,12 +1,15 @@
 <div class="versions">
   <h1>Chart Versions</h1>
-  <div class="version" *ngFor="let version of versions">
+  <div class="version" *ngFor="let version of shownVersions(versions)">
     <span class='number' [class.selected]="isSelected(version)">
       <a [href]="goToVersionUrl(version)" [routerLink]="goToVersionUrl(version)">
         {{ version.attributes.version }}</a>
     </span> -
     <span class='creation-date'>{{ version.attributes.created | date: 'MMM d, y' }}</span>
   </div>
+  <div class="more-link" *ngIf="!showAllVersions">
```

Comment:
I believe that we should hide the link if there are 5 or less versions for a Chart.